chapters/en/slides/module5/module5_01.Rmd

```diff
@@ -0,0 +1,147 @@
+It focuses on the recent declining trend in deaths,
+uses a positive green color,
+and has the bars facing upwards.
+
+Both figures are correct and both use the same data,
+but they convey starkly distinct messages.
+
+This also serves to illustrate that understanding the principles behind designing figures for communication
+does not only help you create effective visualizations,
+it is also of great help when interpreting figures made by others.
+
+---
+
+## Figure design should be adopted to the target audience and situation
+
+<center><img src="/module5/napoleon.png" width="100%"></img></center>
+
+Notes:
+This is an elegant figure
+that goes into excquisite detail
+about how Napoleon's army gradually shrank in numbers as the march against the Russian empire
+in the early 1800s.
+
+The width of the line is the number of soldiers in the army
+and the color represents whether the are marching against Moscow (gold/brown)
+or retreating back to France (black).
+The bottom of the chart shows how the temperature changed during the campaign.
+
+It does take a few seconds to digest the graph when you first see it,
+but once you have familiarized yourself with it,
+you appreciate how you can see in detail the army progress as it moved through Europe.
+
+This is a great visualization when the reader has time to sit down with the plot for a while
+and when you believe your target audience will appreciate this level of detail?
+
+But what if you are presenting this as part of a slide deck in a board meeting
+and only can spend a few seconds describing the plot?
+It would likely be too complicated to digest and the take home message might be lost.
+
+---
+
+## Figure design should be adopted to the target audience and situation
+
+<center><img src="/module5/napoleon-pie.png" width="50%"></img></center>
+
+https://socviz.co/lookatdata.html#lookatdata
+
+Notes:
+In the board meeting,
+this pie chart is likely more effective in communicating the take home message:
+the Napolean army was decimated as a result of this military campaign.
+
+This infomation is simple enough that a pie is effective,
+but in general we should be careful when using pie charts
+
+---
+
+## Including many categories in pie charts makes them hard to interpret
+
+<center><img src="/module5/pie-bar-categories.png" width="70%"></img></center>
+
+https://github.com/glosophy/CatoDataVizGuidelines/blob/master/PocketStyleBook.pdf
+
+Notes:
+When there are too many categories in a pie chart,
+it becomes difficult to compare them with each other.
+Both because the colors can be hard to distinguish
+and because the angle and area of the pie slices is not 
+
+In most cases,
+it is more effective to use position
+as the visual channel to convey a comparison
+instead area and angle.
+
+A bar chart is often a superior alternative to a pie chart,
+and when in doubt chose the bar chart for counts, percentages, and proportions.
+
+
+```python
+
+```
```

Comment:
This slide is a bit odd following the higher-level theme of the preview 5... I think we also already talked about this in an earlier module in detail when we discussed choosing a visual channel. To me this slide deck should be more about audience and polish. Maybe here you could give some examples (at a high level) of the different effective figure design things you are going to dig into in the next few slide decks? Some more time could also be devoted to choosing the level of detail for the audience (and medium!)